src/components/Toolbar.tsx

```diff
@@ -190,7 +190,8 @@ const Toolbar = ({ dark, fontSize, toolbarOverlay, scrollPrioritized, showTopCon
           >
             <span id='left-arrow' className={leftArrowElementClassName}><TriangleLeft width={arrowWidth} height={fontSize} fill='gray' /></span>
             {shortcutIds.map(id => {
-              const { name, svg, exec, isActive } = shortcutById(id)!
+              const { name, svg, exec, isActive, canExecute } = shortcutById(id)!
+              const isToolbarExecutable = isActive ? !isActive || isActive(store.getState) : !canExecute || canExecute(store.getState)
```

Comment:
`!isActive` is always false in the case that `isActive` is true, so it should be omitted.

Reply:
Done in 2a2a12201e

Reply:
Done in 2a2a12201e